### DISCUSSION TITLE
Is there a way to highlight a file in a pull request review based on Codeowner(s) of the file

### DISCUSSION THREAD

**Discussion by Ryang20718** [community/community#58184] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I know there's a way to filter out which files are owned by you when reviewing a PR, but was wondering if there's a way to highlight the code owner(s) automatically for a PR review (e.g at the side or top)

**Comment by the-officialjosh** [comment]
GitHub does not offer a built-in feature to automatically highlight code owners during a pull request (PR) review. However, you can use third-party tools like OctoTree or LinearB to achieve this functionality. OctoTree is a browser extension that allows you to filter and highlight files based on specific criteria, such as code owners. LinearB, on the other hand, is a DevOps platform that integrates with GitHub and provides automated code review requests with highlighted code owners at the top of the PR review.GitHub does not offer a built-in feature to automatically highlight code owners during a pull request (PR) review. However, you can use third-party tools like OctoTree or LinearB to achieve this functionality. OctoTree is a browser extension that allows you to filter and highlight files based on specific criteria, such as code owners. LinearB, on the other hand, is a DevOps platform that integrates with GitHub and provides automated code review requests with highlighted code owners at the top of the PR review.
